.github/workflows/release-please.yml

```diff
@@ -11,6 +11,4 @@ jobs:
       - uses: google-github-actions/release-please-action@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          release-type: python
-          package-name: gyp-next
-          bump-minor-pre-major: true
+          config-file: release-please-config.json
```

Comment:
Why is it useful to put the release context in two separate files?

Reply:
It's not useful, we have no choice as it's mandatory since the version 4 of the action.

Reply:
You could also remove this option since that is also the default.